tests/infrastructure/src/main/java/org/mule/test/infrastructure/profiling/tracing/SpanTestHierarchy.java

```diff
@@ -25,6 +35,20 @@
  */
 public class SpanTestHierarchy {
 
+  public static final String ARTIFACT_TYPE_KEY = "artifactType";
```

Comment:
I would move the original keys to some public location in the case of the errors.